assets/app/views/edit/build-config.html

```diff
@@ -0,0 +1,526 @@
+<project-header class="top-header"></project-header>
+  <project-page>
+    <!-- Middle section -->
+    <div class="middle-section" ng-if="buildConfig">
```

Comment:
This `ng-if` reintroduces the runtime error parsing key/values. It needs to be `ng-show`.